src/logcollector.conf

```diff
@@ -28,11 +28,15 @@
 ## max concurrent connections for fetching RHEV-H logs
 #max-connections=MAX_CONNECTIONS
 
-#
 ###  Database Configuration
-#
+
 ## PostgreSQL database user name
+#
+# The default for pg-user is not hard-coded but is taken from the configuration created by 'engine-setup'
```

Comment:
Actually this is correct for all the credentials below, not just pg-user. The only difference (which is what triggered the process leading to pushing this patch) is that engine-setup's default for pg-user is 'engine', not 'postgres'. So:
1. Perhaps change the text accordingly
2. Perhaps change the comment below to say 'pg-user=engine'.
3. (Also, no need for the empty line after pg-user)

Reply:
done in next commit